Lib/test/test_unittest/testmock/testthreadingmock.py

```diff
@@ -0,0 +1,278 @@
+import time
+import unittest
+import concurrent.futures
+
+from unittest.mock import patch, ThreadingMock, call
+
+
+class Something:
+    def method_1(self):
+        pass
+
+    def method_2(self):
+        pass
```

Comment:
@mariocj89 - when you're doing test coverage for [this](https://github.com/python/cpython/pull/16094/files#r1257929837), please can you tack `  #pragma: no-cover` on these two pass statements as per https://mock.readthedocs.io/en/latest/#backporting-rules?

Reply:
I don't see any other `#pragma: no-cover` in the codebase. Are you sure we should we add this to `cpython`?

Reply:
I don't see any harm, and it makes for less chance of conflicts when backporting stuff in future.
(I _wish_ cpython actually checked code coverage, including in tests, but that's a whole other discussion :-/)

Reply:
https://github.com/python/cpython/pull/106617